orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/api/FrontEndAPI.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.di.orchestration.shared.api;
+
+import com.nimbusds.oauth2.sdk.ErrorObject;
+import com.nimbusds.openid.connect.sdk.Prompt;
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Optional;
+
+import static uk.gov.di.orchestration.shared.helpers.ConstructUriHelper.buildURI;
+
+public class FrontEndAPI {
```

Comment:
I think this is the wrong name. The frontend API exists, but this refers to the frontend

Reply:
Yeah that did occur to me. Not really sure what a ogod name fore it is.